.github/workflows/build.yml

```diff
@@ -42,6 +42,34 @@ jobs:
           go-version: '1.15'
       - name: Install jq
         run: choco install jq
+      - name: BUGFIX - Downgrade Docker CLI if 20.10.0
+        shell: powershell
+        run: |
+          $dockerVersion=(docker version -f '{{.Client.Version}}')
+          if ($dockerVersion -ne "20.10.0") {
+            echo Newer docker version has been released: $dockerVersion
+            echo Remove this step if greater than: 20.10.5
+          }
+
+          curl.exe -o "$env:USERPROFILE\go\bin\docker.exe" -L https://github.com/StefanScherer/docker-cli-builder/releases/download/19.03.14/docker.exe
```

Comment:
This step downgrades the docker CLI from 20.10.0 to 19.03.14 to workaround this [bug](https://docs.docker.com/engine/release-notes/#:~:text=docker%20start%20--attach) that get's fixed in 20.10.5: 
> Revert docker/cli#2960 to fix hanging in docker start --attach
See the `DockerSeedRunAndCopy` comments for more details

Reply:
FWIW, a 20.10.5 CLI could be built from source via `go get` but this would slow down every run